R/qualification-ddi.R

```diff
@@ -217,11 +218,43 @@ generateDDIQualificationDDIPlot <- function(data) {
   ddiPlotConfiguration$export$height <- 2 * 1.2 * (data$plotSettings$height / 96)
   ddiPlotConfiguration$export$units <- "in"
 
+  # Set axis label font size
   ddiPlotConfiguration$labels$xlabel$font$size <- 2 * data$plotSettings$axisFontSize
   ddiPlotConfiguration$labels$ylabel$font$size <- 2 * data$plotSettings$axisFontSize
+
+  # Set axis tick font size
+  ddiPlotConfiguration$xAxis$font$size <- 2 * data$plotSettings$axisFontSize

```

Comment:
Do we agree that this is a quick fix until we have a centralized way for dealing with axis and font size?
@abdullahhamadeh and @pchelle ? 
I don't think that having some magic number spread out everywhere in the code (for each plot) will do us any good in the long run (and even in the short one)

Reply:
Yep.
Generally, multiplying configuration plan sizes by a factor of 2 seems to give outputs of appropriate proportions, comparable to what was produced in matlab.